cromwell-drs-localizer/src/main/scala/drs/localizer/DrsLocalizerMain.scala

```diff
@@ -37,114 +33,38 @@ class DrsLocalizerMain(drsUrl: String,
                        downloadLoc: String,
                        requesterPaysProjectIdOption: Option[String]) extends StrictLogging {
 
-  private final val RequesterPaysErrorMsg = "Bucket is requester pays bucket but no user project provided."
-  private final val ExtractGcsUrlErrorMsg = "No resolved url starting with 'gs://' found from Martha response!"
+  private final val ExtractUriErrorMsg = "No access URL nor GCS URI starting with 'gs://' found in Martha response!"
 
-  def getGcsDrsPathResolver: IO[GcsLocalizerDrsPathResolver] = {
+  def getDrsPathResolver: IO[LocalizerDrsPathResolver] = {
     IO {
       val drsConfig = DrsConfig.fromEnv(sys.env)
-      new GcsLocalizerDrsPathResolver(drsConfig)
+      new LocalizerDrsPathResolver(drsConfig)
```

Comment:
Very very ToL: While renaming, should `LocalizerDrsPathResolver` be called `DrsLocalizerDrsPathResolver` to match `DrsLocalizerMain`?

See also: http://java.metagno.me/

Or maybe `DrsLocalizerMain` just becomes something like `Main`. 🤷